### ISSUE TITLE
Error ID: 43997dd244c04e9c96199d94a8b170bc

### ISSUE THREAD

**Issue by henrymgarrett** [stacktape/stacktape#84]
Hey all,  I'm trying to deploy an npm workspaces monorepo that uses Typescript project references for code sharing.
When deploying normally (no language specific inputs to the stacktape buildback), I received the following error:
Config:
```
          packaging:
            type: stacktape-image-buildpack
            properties:
              entryfilePath: ./packages/api/src/index.ts
```
Error
```
PACKAGING: Failed to build code at ./packages/api/src/index.ts
          Can't resolve '@receipts-xyz/service' (imported from ./packages/api/src/resolvers/index.ts:4:28).
        There might be 2 reasons this has happened:
          1. These dependencies are not installed.
          2. These dependencies are imported (required) conditionally and your application does not need them, but Stacktape still tries to statically bundle them.
```

So I tried to include the language specific parameters:
```
          packaging:
            type: stacktape-image-buildpack
            properties:
              entryfilePath: ./packages/api/src/index.ts
              # The below causes an error for some reason.  TBD
              languageSpecificConfig:
                outputModuleFormat: 'esm'
                tsConfigPath: './packages/api/tsconfig.json'
```
And receive the following error:
```
UNEXPECTED_ERROR] An unexpected error occurred. Last captured event: PACKAGE_ARTIFACTS.
[HINT] This error has been anonymously reported to our error monitoring service with id 43997dd244c04e9c96199d94a8b170bc.
```

The repo structure is like so:

/packages
  /api
    package.json
    tsconfig.json -> this references the service directory tsconfig.json via `references`
  /service
    package.json
    tsconfig.json
tsconfig.json -> root config that all package tsconfig.json's inherit from

**Comment by matuscongrady** [comment]
Hello Henry,

The issue you're encountering is (most likely) related to Stacktape not being able to parse your `tsconfig.json` due to the fact that it includes comments. This has been resolved in version `2.5.5` and should now work properly.

That being said, Stacktape should be able to package (resolve) imports from your workspaces automatically. We've created example projects:

- Monorepo with pnpm: https://github.com/stacktape/starter-monorepo-typescript-pnpm
- Monorepo with npm or yarn: https://github.com/stacktape/starter-monorepo-typescript

Please also make sure that you've run your initial installation using a proper package manager, and that a lockfile for your package manager of choice has been generated. 

Note that Stacktape performs package installation before packaging workloads (containers and lambda functions), that use stacktape-native buildpack. It deduces the correct package manager based on the lockfile being present in the repository, and defaults to `npm` if none has been found.

**Comment by simi-obs** [comment]
Closing as resolved
